docassemble/ALToolbox/display_template.py

```diff
@@ -0,0 +1,35 @@
+import re
+from .copy_button import *
+
+def display_template(template, scrollable=True, collapse=False, copy=False, class_name='bg-light'):   
+  # 1. Initialize    
+  if scrollable:
+    scroll_class = 'scrollable-panel'
+    adjust_height = ''
+  else:
+    scroll_class = ''
+    adjust_height = f'onmouseover="this.style.height = (this.scrollHeight) + \'px\';"'       
+
+  class_name = ' ' + class_name.strip()  
```

Comment:
It looks like we're passing this to other functions. If so, I think we're sticking with a "black box" model of programming. That is, this function "doesn't know" how the next function will use this text. We should just hand down the plain text and let the future worry about handling whitespace.

Alternatively, if we really want to handle whitespace here (which I'd love to discuss), we should do it with the other strings as well, like `adjust_height`.

Reply:
Sorry @BryceStevenWilley, you didn't add line 13 above with blank space, my memory got mixed up. That line was carried over from `collapse_template()`